src/org/opendatakit/briefcase/ui/PullTransferPanel.java

```diff
@@ -266,7 +266,7 @@ public void actionPerformed(ActionEvent arg0) {
     });
 
     formTransferTable = new FormTransferTable(
-        btnSelectOrClearAllForms, FormStatus.TransferType.GATHER, btnTransfer, btnCancel);
+        btnSelectOrClearAllForms, FormStatus.TransferType.GATHER, btnTransfer, btnCancel, null);
```

Comment:
Remove all the changes on this file and center your attention on `OriginActionListener.actionPerformed(ActionEvent e)`. Once we know that the user has set the source settings, we can call the table's `setSourceSelected(boolean)` method.